src/resource.c

```diff
@@ -455,6 +455,12 @@ coap_find_attr(coap_resource_t *resource,
   return NULL;
 }
 
+coap_str_const_t *coap_attr_get_value(coap_attr_t *attr) {
```

Comment:
Please insert a line break after the return type.

Reply:
Corrected